crypto/ec/curve448/curve448.c

```diff
@@ -586,9 +586,15 @@ static int recode_wnaf(struct smvt_control *control,
             int32_t delta = odd & mask;
 
             assert(position >= 0);
-            assert(pos < 32);       /* can't fail since current & 0xFFFF != 0 */
             if (odd & (1 << (table_bits + 1)))
                 delta -= (1 << (table_bits + 1));
+            /*
+             * Coverity gets confused by the value of pos, thinking it might be
+             * 32. This would require current & 0xFFFF to be zero which isn't
```

Comment:
You might use 2 spaces after the period here, like it is done in the following line.
Well its GNU style, so you don't have to do it....

Reply:
We're not at all consistent about spaces after full stops.
I've added the space.